gallery/src/data-visualization/ReactVisDonutChartPage.tsx

```diff
@@ -0,0 +1,180 @@
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>colorType</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Override to let React-vis know that the colors provided are literal color values
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>innerRadius</NxTableCell>
+              <NxTableCell>number (in pixels)</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                If radius is not set on the data then this can be used to set the innerRadius for all of the rows.
+                This can be useful for building donut charts.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>labelsRadiusMultiplier</NxTableCell>
+              <NxTableCell>number</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                How far the labels should be from the center of the chart as a function of the radius of the chart.
+                If not specified, the default value of 1.1 is used (slightly outside of the chart).
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>onValueMouseOut</NxTableCell>
+              <NxTableCell>function</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                This handler is triggered either when the user's mouse leaves a mark. The handler passes two arguments,
+                the corresponding datapoint and the actual event.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>onValueMouseOver</NxTableCell>
+              <NxTableCell>function</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                This handler is triggered either when the user's mouse enters a mark. The handler passes two arguments,
+                the corresponding datapoint and the actual event.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>padAngle</NxTableCell>
+              <NxTableCell>number/function</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                The padding to be applied between arcs.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>radius</NxTableCell>
+              <NxTableCell>number</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                The distance between the origin and the outside of the arc. This values is scaled linearly by default
```

Comment:
How else would it scale?

Reply:
> How else would it scale?

Again, no details on it.
![image](https://user-images.githubusercontent.com/7110862/126370022-5f7e22e8-a94a-4c75-b8b2-038c953ab6c5.png)

Reply:
> Again, no details on it.

I'd be in favor of dropping that note from our docs then.  The default is what one would expect, so we can leave it to the react-vis docs to explain whatever other possibilities there are.